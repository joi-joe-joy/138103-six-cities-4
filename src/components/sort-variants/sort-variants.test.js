import React from "react";
import renderer from "react-test-renderer";
import SortVariants from "./sort-variants.jsx";

const children = <div/>;

it(`Render CitiesPlaces correctly`, () => {
  const tree = renderer
    .create(
        <SortVariants>
          {children}
        </SortVariants>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
