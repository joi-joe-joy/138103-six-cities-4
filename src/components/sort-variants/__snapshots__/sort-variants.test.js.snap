// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CitiesPlaces correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <div />
</form>
`;
